### DISCUSSION TITLE
testvideosrc like test pattern??

### DISCUSSION THREAD

**Discussion by yumnycheesecakes** [savonet/liquidsoap#3397] [General]
Hi everyone, is there a test pattern like testvideosrc?? I need it. Thanks.

**Comment by smimram** [comment]
Yes, you have the [video.testsrc](https://www.liquidsoap.info/doc-dev/reference.html#video.testsrc) operator, as well as [video.testsrc.ffmpeg](https://www.liquidsoap.info/doc-dev/reference.html#video.testsrc.ffmpeg).

**Reply by yumnycheesecakes** [reply]
Thanks. That is very helpful because my web-channel runs on a specific schedule and when it's night, it's going to switch to that pattern.
